bitrise.yml

```diff
@@ -138,7 +222,7 @@ workflows:
         title: Pull artifacts
         inputs:
         - verbose: true
-        - artifact_sources: stage_1.*
+        - artifact_sources: stage_1.configure_build_fennec_xcuitest
```

Comment:
We only need the artifacts generated in stage1 in this workflow so that the UI tests could run.